jlm/llvm/opt/alias-analyses/PointerObjectSet.hpp

```diff
@@ -701,6 +703,36 @@ class PointerObjectConstraintSet final
   using ConstraintVariant =
       std::variant<SupersetConstraint, StoreConstraint, LoadConstraint, FunctionCallConstraint>;
 
+  enum class WorklistSolverPolicy
+  {
+    LRF,
```

Comment:
Could we spell them out? While I certainly know what Fifo and Lifo means, I had to think now a bit about LRF (least recently fired?) Also, could we put the papers references as documentation behind them such that we have some bread crumbs here.

Reply:
@phate I changed them to be written out, and added a reference to the paper, and to the corresponding `jlm::util::___Worklist` classes.